packages/starlight/schema.ts

```diff
@@ -38,6 +38,14 @@ export function docsSchema() {
        */
       description: z.string().optional(),
 
+      /**
+       * The order of this page in the navigation.
+       * Pages are sorted by this value in ascending order. Then by slug.
+       * If not provided, pages will be sorted alphabetically by slug.
+       * If two pages have the same order value, they will be sorted alphabetically by slug.
+       */
+      order: z.number().optional(),
```

Comment:
Can we move this under a `sidebar` key? I think we’re very likely to want more things here, so this would allow for a nice grouping and I think also clarifies a bit what this is the order of. For example frontmatter might end up looking like this eventually:
```yml
title: Page title may be a bit longer
sidebar:
  order: 1
  label: Short nav title
```

Reply:
Yeah, I agree that is better. I will make the required changes

Reply:
Great — thank you! Took the liberty of resolving conflicts and updating your branch, so you should now have test files on here too.